ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/util/BlockBlobSidecarsTrackersPoolImpl.java

```diff
@@ -127,6 +131,11 @@ public class BlockBlobSidecarsTrackersPoolImpl extends AbstractIgnoringFutureHis
 
   private final BlockImportChannel blockImportChannel;
 
+  private final boolean isSuperNode;
+  private final AtomicBoolean isActiveSuperNode = new AtomicBoolean(false);
+  private final Consumer<List<DataColumnSidecar>> dataColumnSidecarPublisher;
+  private final Supplier<KZG> kzgSupplier;
```

Comment:
nit: I would better group the fields passed to constructor together

Reply:
yeah, it looks better now